cmd/kubeadm/app/preflight/checks.go

```diff
@@ -119,6 +121,7 @@ type FirewalldCheck struct {
 	ports []int
 }
 
+// Check validates if the firewall is enabled and active.
```

Comment:
firewalld is enabled and active. If it is, warn the user that there may be problems if not actions are taken

Reply:
done